network-api/networkapi/urls.py

```diff
@@ -46,12 +49,11 @@
 
     # network API routes:
 
-    url(r'^api/people/', include('networkapi.people.urls')),
+    url(r'^api/campaign/', include('networkapi.campaign.urls')),
```

Comment:
why do we still need this url set?

Reply:
because the campaign app _does_ still do work for us. It's the POST handler for petition data to be forwarded to gsheets/salesforce via SQS